app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
```

Comment:
Code Style Issue: According to the checklist, it's better to use parentheses `()` while importing multiple modules or classes. So, you should consider changing the import statement to `from app.errors import (NotWearingMaskError, VaccineError)`